### ISSUE TITLE
Webdev support

### ISSUE THREAD

**Issue by jonas-jonas** [eclipse/dartboard#110]
Dart projects that are supposed to be run in the browser, need to be launched using the [`webdev`](https://dart.dev/tools/webdev) CLI tool.
This needs to be supported in the launch configurations.

For this we should somehow detect if the project depends on `build_web_compilers`. This can either done
1. by parsing the `pubspec.yaml` file _or_
2. by looking at the output of `pub deps -s compact` (also relevant for #109)

I'm not sure yet, what would be the better option.

If the detection detects such a project, there should be another **Run as** entry titled **Run as webdev application**, which launches the `webdev` cli in the current project. Also; there are two commands relevant here `webdev serve` and `webdev build`. Both should be supported.

**Comment by lak-proddev** [comment]
@jonas-jonas  I am interested to work on it. If you are not started yet.

**Comment by jonas-jonas** [comment]
No, feel free to start work on it.

Lakshminarayana Nekkanti <notifications@github.com> schrieb am Mi., 31.
Juli 2019, 07:40:

**Comment by jonas-jonas** [comment]
Also note, that webdev should be invoked using the `pub global run` command. See  the [stagehand](https://github.com/eclipse/dartboard/tree/stagehand) branch for a how to as it should be the same commands.

**Comment by lak-proddev** [comment]
I think these are the sub-tasks.

- [ ] Detect Webdev support should be enabled for the project.
- [ ] Provide Launch configuration
     - [ ] webdev serve support
     - [ ] webdev build support

**Comment by jonas-jonas** [comment]
Yes, looks good to me. Please use https://github.com/eclipse/dartboard/blob/master/org.eclipse.dartboard/src/org/eclipse/dartboard/util/PubUtil.java when instantiating a new ProcessBuilder so you have the PUB_ENVIRONMENT variable set.
